script/States/App/Examples/HmGui/ScrollArea.lua

```diff
@@ -16,13 +16,13 @@ function Test:scrollArea()
 
     self.fading = Gui:checkbox("Fading", self.fading)
 
-    Gui:setPropertyBool(GuiProperties.ScrollAreaScrollbarVisibilityFading, self.fading)
-    Gui:setPropertyColor(GuiProperties.ScrollAreaScrollbarKnobColor, Color(1, 0, 0, 1))
-    Gui:setPropertyBool(GuiProperties.ScrollAreaHScrollShow, false)
-    Gui:setPropertyColor(GuiProperties.BackgroundColor, Color(0, 0, 1, 1))
+    Gui:setProperty(GuiProperties.ScrollAreaScrollbarVisibilityFading, GuiPropertyValue.FromBool(self.fading))
```

Comment:
this looks like a case where an extension would be useful. Make a table of the properties and their expected values and then wrap the GuiPropertyValue.FromX() in the extension. I don´t think its a good idea to have this bestowed upon the regular scripter.

Reply:
Yeah, I was also thinking about the extensions.. Will add it in one of the next commit